test/javascripts/components/chat-channel-title-test.js

```diff
@@ -184,5 +185,33 @@ discourseModule(
         );
       },
     });
+
+    componentTest("unreadIndicator", {
+      template: hbs`{{chat-channel-title channel=channel unreadIndicator=unreadIndicator}}`,
+
+      beforeEach() {
+        const channel = fabricate("chat-channel", {
+          chatable_type: CHATABLE_TYPES.directMessageChannel,
+        });
+
+        const state = {};
+        state[channel.id] = {
+          unread_count: 1,
+        };
+        set(this.currentUser, "chat_channel_tracking_state", state);
```

Comment:
Why `set(this.currentUser)` and not `this.currentUser.set()`? 😃

Reply:
no specific reason, I changed to this.currentUser.set